myapp/config/locales/ja.yml

```
@@ -72,6 +72,8 @@ ja:
       sort:
         latest: "新しい順"
         oldest: "古い順"
+        closest_deadline: "期日の近い順"
+        longest_deadline: "期限の長い順"
```

When I hear "long deadline", I imagine a very long period of time (like a year) 💭 
遠い seems more natural, but nice try 👍 
(by the way, in English, longest deadline is natural? 👀  )

```suggestion
        closest_deadline: "期日の近い順"
        longest_deadline: "期限の遠い順"
```

---

thank you.
yep `far_deadline` would be more natural, i think!